doc/source/development/contributing.rst

```diff
@@ -208,7 +208,7 @@ We'll now kick off a three-step process:
 
    # Build and install pandas
    python setup.py build_ext --inplace -j 4
-   python -m pip install -e .
+   python -m pip install -e --no-build-isolation .
```

Comment:
```
$ python -m pip install -e --no-build-isolation .
ERROR: --no-build-isolation is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).
(pandas-dev)
simon@DESKTOP-F063QUA MINGW64 ~/OneDrive/code/pandas-simonjayhawkins (master)
$ python -m pip install -e . --no-build-isolation
Obtaining file:///C:/Users/simon/OneDrive/code/pandas-simonjayhawkins
    Preparing wheel metadata ... done
```

Reply:
That's a typo, the `.` needs to come after the `-e` -> https://github.com/pandas-dev/pandas/pull/28445